src/Routing/Route/Route.php

```diff
@@ -146,7 +146,7 @@ public function compiled()
      */
     public function compile()
     {
-        if (!empty($this->compiledRoute)) {
+        if (!empty($this->_compiledRoute)) {
```

Comment:
Yes. But at the moment, I have no objection to use `empty()` an existing property :)
Thank you for finding this issue, @Phillaf.